torch/csrc/jit/passes/onnx.cpp

```diff
@@ -375,16 +376,33 @@ void NodeToONNX(
 
     WithInsertPoint insert_point_guard(new_block);
     WithCurrentScope scope_guard(*new_block->owningGraph(), op->scope());
-    // Call the symbolic function
-    // Use a little trampoline function so we can give good error messages
-    // upon argument mismatch
-    py::object opset_version = onnx_symbolic.attr("_export_onnx_opset_version");
-    onnx_registry.attr("register_op")(
-        op->name(), pyobj.attr("symbolic"), "", opset_version);
-    py::object raw_output = onnx.attr("_run_symbolic_method")(
-        op->name(), pyobj.attr("symbolic"), py_symbolic_args);
-
-    processSymbolicOutput(op->name(), op, raw_output);
+
+    if (py::hasattr(pyobj, "symbolic")) {
+      // Call the symbolic function
+      // Use a little trampoline function so we can give good error messages
+      // upon argument mismatch
+      py::object opset_version = onnx_symbolic.attr("_export_onnx_opset_version");
+      onnx_registry.attr("register_op")(
+          op->name(), pyobj.attr("symbolic"), "", opset_version);
+      py::object raw_output = onnx.attr("_run_symbolic_method")(
+          new_block->owningGraph(), op->name(), pyobj.attr("symbolic"), py_symbolic_args);
+
+      processSymbolicOutput(op->name(), op, raw_output);
+    } else {
+      TORCH_INTERNAL_ASSERT(PyObject_IsTrue(is_registered_op.ptr()));
+      Node* n = static_cast<Node*>(op);
+      n->s_(attr::name, op->name());
+      // Call symbolic function
+      py::object raw_output = onnx.attr("_run_symbolic_function")(
```

Comment:
Maybe we need to call another function such as `onnx.attr("_run_python_op")`?

Reply:
That won't be necessary. The design is to funnel all symbolic function calls through `_run_symbolic_function`.